setup.py

```diff
@@ -27,5 +27,15 @@
     entry_points={
         'console_scripts': ['cibyl = cibyl.cli.main:main',
                             'ci = cibyl.cli.main:main']
-    }
+    },
+    install_requires=[
+        'pyyaml>=5.0',
+        'requests~=2.27.1',
+        'elasticsearch~=7.17.1',
+        'colorlog~=6.6.0',
+        'urllib3~=1.26.9',
+        'rfc3987~=1.3.8',
+        'overrides~=6.1.0',
+        'GitPython~=3.1.18',
+    ]
```

Comment:
Something like this may be better: https://stackoverflow.com/a/16624700/19144577

Reply:
Yes, I also think this would be best. We have been using a 'requirements.txt' thus far and I think it would be easier if setup.py just copied the contents from that file.